test/e2e/framework/test_context.go

```diff
@@ -277,6 +277,8 @@ type NodeTestContextType struct {
 	ExtraEnvs map[string]string
 	// StandaloneMode indicates whether the test is running kubelet in a standalone mode.
 	StandaloneMode bool
+	// CriProxyEnabled indicates whether enable CRI API proxy for failure injection.
+	CriProxyEnabled bool
```

Comment:
Can you please include the test case example in this PR as well. This will be easier to reason about usability of APIs when we see the test example

Reply:
done